src/App.js

```diff
@@ -34,8 +38,24 @@ function App() {
           <Route path="/signup" element={<Signup />} />
           <Route path="/look" element={<Look />} />
           <Route path="/record" element={<WithLogin children={<Record />} />} />
-          <Route path="/create" element={<WithLogin authority="ROLE_TEACHER" children={<Create />} />} />
-          <Route path="/application/:id" element={<WithLogin authority="ROLE_TEACHER" children={<ApplicationManagement />} />} />
+          <Route
+            path="/create"
+            element={
+              <WithLogin authority="ROLE_TEACHER" children={<Create />} />
+            }
+          />
+          {/* Application 페이지 */}
+          <Route path="/application/:id" element={<ApplicationPage />} />
```

Comment:
```suggestion
          <Route path="/application/:id" element={<ApplicationPage />} />
```

주석 지워주세욤
그리고 이거 commit suggestion 버튼 누르면 그냥 커밋 됨니다